docs/api-docs/channels/building-channel-apps.md

```diff
@@ -26,7 +40,7 @@ This article is a comprehensive guide on the foundations of integrating [Channel
 
 ## Getting started
 
-Here's a few things you'll need to create channel apps.
+Here are a few things you'll need to create channel apps.
```

Comment:
```suggestion
Here are a few things you will need to create channel apps.
```

Reply:
Do we have an official stance on contractions? I changed "here's" because it references multiple things, and one typically wouldn't use the singular "here're" in writing. "You'll" feels conversational enough to remain, though.

Reply:
We do not have an official stance on contractions. I will defer to @uwikaiddi on this.